dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
@bkardell I don't think this polyfill should give authors abilities they won't have with the actual `:focus-ring` psuedo. Therefore, once actual support for `:focus-ring` lands in browsers the only change authors need to make to their CSS is changing their rules from `.focus-ring` to `:focus-ring`. Hence, more with the grain.